catalyst_voices/packages/libs/catalyst_cardano_serialization/lib/src/builders/types.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:catalyst_cardano_serialization/catalyst_cardano_serialization.dart';
+
+/// Represents the valid range for coin values as a tuple of minimum and
+/// maximum [Coin] values.
+typedef ValidRange = (Coin, Coin);
```

Comment:
For all the typedefs defined in here I would add a name alongside the type to all function parameters.

I.e.

```suggestion
typedef ValidRange = (Coin minInclusive, Coin maxInclusive);
```

Reply:
as not every parameter goal can be deducted just from reading the code here

Reply:
done